intake_parquet/source.py

```diff
@@ -106,5 +106,32 @@ def _to_dask(self):
         self._load_metadata()
         return self._df
 
+    def to_spatial(self):
+        """
+        Create a datashader spatial object from the parquet data
```

Comment:
I think this needs a lot more description, and certainly a link to wherever the general idea is described (e.g., how do you make these special parquet datasets?)

Reply:
Agreed.